ui/packages/i18n_opps/CNfinder.js

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+const babelParser = require('@babel/parser');
+const babelTraverse = require('@babel/traverse').default;
+const babelGenerator = require('@babel/generator').default;
+const { Command } = require('commander');
+const crypto = require('crypto');
+const { exec } = require('child_process');
+
+const program = new Command();
+
+// 配置命令行选项
+program
```

Comment:
1. the process of parse options does not mentioned in the detail design
2. can we support config file as optioins, like `-c xx.config.js`